tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -235,6 +235,17 @@ mod generated_constants {
     fn monster_file_extension() {
         assert_eq!("mon", my_game::example::MONSTER_EXTENSION);
     }
+
+    #[test]
+    fn enum_constants() {
+        assert_eq!(my_game::example::ENUM_MIN_COLOR, 1);
+        assert_eq!(my_game::example::ENUM_MAX_COLOR, 8);
+        assert_eq!(my_game::example::ENUM_VALUES_COLOR, [
+            my_game::example::Color::Red,
+            my_game::example::Color::Green,
+            my_game::example::Color::Blue,
+        ]);
```

Comment:
Is it reasonable to also include the other generated public constants (basically, all the ones that are in the diff for this PR) in the test? If so, please do that.

Reply:
Done.